### PR TITLE
chore(assistant): Better type comments

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/components/assistant/types.tsx [hunk 1]
```diff
@@ -38,12 +56,21 @@ type BaseGuide = {
...
 
 export type GuidesContent = BaseGuide[];
 
 export type GuidesServerData = {
+  /**
+   * Guide key
+   */
   guide: string;
+  /**
+   * Has the guide been seen
```
Comment (priscilawebdev) [inline static/app/components/assistant/types.tsx:73]: shouldn't we use the same comment here?

```
/**
   * Has this guide already been seen?
   */
```

File: static/app/components/assistant/types.tsx [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+   * Disables dismissal
+   */
   cantDismiss?: boolean;
+  /**
+   * Label for the dismiss button
+   */
   dismissText?: string;
   hasNextGuide?: boolean;
+  /**
+   * Label on the next button
```
Comment (priscilawebdev) [inline static/app/components/assistant/types.tsx:16]:
```suggestion
   * Label for the next button
```

File: static/app/components/assistant/types.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/components/assistant/types.tsx:73]: yeah will fix these two small ones, have another PR that depends on these changes tho :)

Comment (evanpurkhiser) [reply static/app/components/assistant/types.tsx:73]: Oh, no approval, i'll make the changes then :)

**Review by davidenwang — APPROVED** [review]

**Pull request merged**